docs/server_notices.md

```diff
@@ -0,0 +1,68 @@
+Server Notices
+==============
+
+'Server Notices' are a new feature introduced in Synapse 0.30. They provide a
+channel whereby server administrators can send messages to users on the server.
+
+They are used as part of the communication of the server polices (see
```

Comment:
drop the 'the'?

Reply:
"part of communication of the server polices" ? really?

"part of communicating" maybe?